<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab { height: 60px;}
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>MAIN</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_reconnect">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ReConnect</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_communication">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>250</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>No Communication</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>210</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>INPUTS</string>
      </property>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>INPUT 1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_input_1">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>--</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>INPUT 2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_input_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>--</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>INPUT 3</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_input_3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>--</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>INPUT 4</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_input_4">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>--</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>300</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>RELAYS</string>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>280</width>
         <height>40</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RELAY 1</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_relay1_on">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ON</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_relay1_off">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>OFF</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>280</width>
         <height>40</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RELAY 2</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_relay2_on">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ON</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_relay2_off">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>OFF</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>280</width>
         <height>40</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RELAY 3</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_relay3_on">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ON</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_relay3_off">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>OFF</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>280</width>
         <height>40</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RELAY 4</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_relay4_on">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ON</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_relay4_off">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>5</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>OFF</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>280</width>
         <height>40</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>280</width>
         <height>40</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>210</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>RAW DATA</string>
      </property>
      <widget class="QGroupBox" name="groupBox_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_adc_raw">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>5</y>
          <width>170</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>--</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_17">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>250</y>
        <width>530</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>VOLTAGE OUTPUT</string>
      </property>
      <widget class="QSlider" name="horizontalSlider_voltage">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>501</width>
         <height>29</height>
        </rect>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_voltage">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionDeneme">
   <property name="text">
    <string>deneme</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
